### ISSUE TITLE
Xamarin version of binaries

### ISSUE THREAD

**Issue by kevinjpetersen** [openalpr/openalpr#447]
I'm trying to integrate the binaries into a Xamarin Android app, however the .NET version does not compile with the ARM processor.

Is it possible to build OpenALPR library as .NET for the ARM processor for Xamarin?
If not, are there other ways to integrate OpenALPR into a Xamarin project?

**Comment by hnabbasi** [comment]
Any update on this?

**Comment by eggeggss** [comment]
Good idea

**Comment by homerokzam** [comment]
Any news?

**Comment by kevinjpetersen** [comment]
Yes! I've been working hard on it, and I've done it! :) Here it is! (Currently only supports Android) https://github.com/kevinjpetersen/openalpr-xamarin

**Comment by eggeggss** [comment]
amazing man ~

**Comment by eggeggss** [comment]
Excuse me~why is 404 not found?
